package.json

```diff
@@ -59,8 +59,11 @@
   },
   "license": "MIT",
   "scripts": {
-    "build": "tsc -p tsconfig.json",
-    "clean": "rm -rf ./.nyc_output ./node_modules/.cache ./coverage ./lib",
+    "build": "yarn build-esm && yarn build-cjs && yarn rename-entrypoints && yarn lint --fix",
+    "build-esm": "tsc -p tsconfig.esm.json && echo '{\"type\":\"commonjs\"}' > cjs/package.json",
+    "build-cjs": "tsc -p tsconfig.cjs.json && echo '{\"type\":\"module\"}' > esm/package.json",
+    "rename-entrypoints": "mv cjs/stripe.cjs.worker.js cjs/stripe.worker.js && mv cjs/stripe.cjs.node.js cjs/stripe.node.js && mv esm/stripe.esm.worker.js esm/stripe.worker.js && mv esm/stripe.esm.node.js esm/stripe.node.js",
```

Comment:
Is this too hacky? I struggled to find a better alternative that works, unless we're OK with keeping the name `cjs/stripe.cjs.node.js` or something similarly verbose

Reply:
> We could also get rid of this logic if we're OK with keeping the name cjs/stripe.cjs.node.js 

Repetitive names don't bother me.